components/Article/AuthorList/Author/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { injectIntl, WrappedComponentProps } from 'react-intl';
+import styles from './index.module.scss';
+import authorPlaceholderImg from './placeholder-img.png';
```

Comment:
This image should be placed in the public folder and be loaded with `next/image`. Also, if possible, an SVG would be preferred.

Reply:
(I know you're using background attributes, but I rather using `next/image` as you can also use `placeholder` images with it.

Reply:
In case of unavailable users, placeholder is not being displayed as fallback, Is this fine?

![image](https://user-images.githubusercontent.com/42532987/232547750-644c129f-c7b2-4e43-ba1f-966554b82f64.png)

Reply:
when using bg attribute, placeholder image will be rendered in case 404 authors

Reply:
I think on `next/image` you can set:

```jsx
onError={() => setImageSrcToPlaceHolder()}
```

Pretty much if the image gives 404 or any error you set the source to the fallback one 👀

Reply:
Note that in that case the `src` must become a `useState`

Reply:
@shanpriyan did it work well?

Reply:
Yes, it did

Reply:
![image](https://user-images.githubusercontent.com/42532987/232864672-d14013f2-b547-4280-b5ad-6de6a0ebfa47.png)